src/components/voting/votingRow.js

```diff
@@ -21,7 +22,7 @@ class VotingRow extends React.Component {
   render() {
     const props = this.props;
     const { data } = props;
-    return (<TableRow {...props} className={`${styles.row} ${setRowClass(data)}`}>
+    return (<TableRow {...omit(props, ['data'])} className={`${styles.row} ${setRowClass(data)}`}>
```

Comment:
Good 👍 But you can simply use `const {data, ...other} = props;` to omit the property and pass to component `{...other}` instead of using `lodash`

Reply:
Good catch, it's ok now